concent_api/core/views.py

```diff
@@ -586,3 +719,20 @@ def get_file_status(file_transfer_token_from_database: message.concents.FileTran
         return False
     else:
         raise exceptions.UnexpectedResponse()
+
+
+def decode_key(key: str) -> bytes:
+    return b64decode(key.encode('ascii'), validate = True)
+
+
+def decode_client_public_key(request):
+    return decode_key(request.META['HTTP_CONCENT_CLIENT_PUBLIC_KEY'])
+
+
+def decode_other_party_public_key(request):
+    if 'HTTP_CONCENT_OTHER_PARTY_PUBLIC_KEY' not in request.META:
+        raise Http400('Missing Other-Party-Public-Key HTTP when expected.')
+    try:
+        return decode_key(request.META['HTTP_CONCENT_OTHER_PARTY_PUBLIC_KEY'])
```

Comment:
This was supposed to be called `Concent-Requestor-Public-Key`.

Reply:
But it is not always requestor key, as for example in second use case.

Reply:
OK.

Reply:
One more thing. If something like has to change during implementation please always drop a comment in the original issue so that we can discuss it. The pull request should implement exactly what the issue says.